department-of-veterans-affairs/vets-api
# Remove usage of edu_benefits_stem_scholarship

## Description of change

This feature flag is causing issues in production with the usage of "Logged in users" Flipper feature

Adding back in unit test data from https://github.com/department-of-veterans-affairs/vets-api/pull/4106/files so that 1995s and 10203 show up in YTD reports

Also removes the flag from disabling after submit emails for 10203

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000
http://sentry.vfs.va.gov/share/issue/38efaaeec2d24185b4d75036b3ef5712/

## Things to know about this PR

To fully test requires it to be in staging as adding emails to "Logged in Users" results in a forbidden error locally

To test on staging
1. Add emails to "Logged in Users" for `edu_benefits_stem_scholarship`
1. Submit a https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction with an added email
1. Submit a https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction without an added email

To test locally
1. Submit a localhost:3001/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/
1. disable `edu_benefits_stem_scholarship`
1. Submit a localhost:3001/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/

Expected that both submissions succeed when testing in local or staging cases
